packages/components/src/DatePicker/utils/getLocale.ts

```diff
@@ -1,21 +1,152 @@
 import type { Locale } from "date-fns"
-import { enAU, enUS, frCA } from "date-fns/locale"
-import { StringSuggestions } from "~types/StringSuggestions"
+import {
+  arSA,
+  bg,
+  cs,
+  cy,
+  da,
+  de,
+  el,
+  enGB,
+  enUS,
+  es,
+  et,
+  fi,
+  fr,
+  frCA,
+  he,
+  hi,
+  ht,
+  hu,
+  id,
+  it,
+  ja,
+  km,
+  ko,
+  lt,
+  lv,
+  ms,
+  nb,
+  nl,
+  pl,
+  pt,
+  ptBR,
+  ro,
+  ru,
+  sk,
+  sr,
+  sv,
+  th,
+  tr,
+  uk,
+  vi,
+  zhCN,
+  zhTW,
+} from "date-fns/locale"
 
 // Ensure you update the storybook DATE_PICKER_SUPPORTED_LOCALES arg options when updating DatePickerSupportedLocales.
-export type DatePickerSupportedLocales = StringSuggestions<
-  "en-US" | "en-AU" | "fr-CA"
->
+export type DatePickerSupportedLocales = string
 
 export const getLocale = (locale: DatePickerSupportedLocales): Locale => {
   switch (locale) {
-    case "en-AU":
-      return enAU
+    case "zh":
+      return zhCN
+    case "zh-TW":
+      return zhTW
+    case "da":
+      return da
+    case "nl":
+      return nl
+    case "en":
```

Comment:
This would change current behaviour for "en" which now uses AU and would start using US. Can we map "en" to "en-GB"?

Reply:
Hmm...I suppose that's true, but would it not be strange for `en` to be `en-US` for everything except the datepickers, therefore making that previously a bug?
I'll change it anyway; it will be easy to drop it again if we want.

Reply:
Yeah it is strange, but it is how it is today. We're intentionally keeping the current unexpected behaviour instead of replacing it with different unexpected behaviour (for AU users). We'll get to this when we properly plan for uplifting the DatePickers or i18n as whole.